var/spack/repos/builtin/packages/justbuild/package.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+
+from spack.package import *
+
+
+class Justbuild(Package):
+    "just, a generic build tool"
+
+    git = "https://github.com/just-buildsystem/justbuild.git"
+
+    tags = ["build-tools"]
+
+    executables = ["^just$", "^just-mr$"]
```

Comment:
Are both of these always installed? Only one (that is always installed) is needed for https://spack.readthedocs.io/en/latest/packaging_guide.html#making-a-package-discoverable-with-spack-external-find.

Reply:
If your goal is to make sure the executables are installed, you could add sanity checks (see https://spack.readthedocs.io/en/latest/packaging_guide.html#adding-sanity-checks).

Reply:
thanks @tldahlgren 

I implemented your suggestions. Would you mind to take a second look at it?

both the executables are always installed